cms/templates/widgets/sock.html

```diff
@@ -18,41 +18,7 @@
         <h2 id="sock-heading" class="title sr-only">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h2>
       <div class="support">
         <%!
-        from django.conf import settings
-
-        partner_email = settings.PARTNER_SUPPORT_EMAIL
-
-        links = [{
-            'href': 'http://docs.edx.org',
-            'sr_mouseover_text': _('Access documentation on http://docs.edx.org'),
-            'text': _('edX Documentation'),
-            'condition': True
-        }, {
-            'href': 'https://open.edx.org',
-            'sr_mouseover_text': _('Access the Open edX Portal'),
-            'text': _('Open edX Portal'),
-            'condition': True
-        }, {
-            'href': 'https://www.edx.org/course/overview-creating-edx-course-edx-edx101#.VO4eaLPF-n1',
-            'sr_mouseover_text': _('Enroll in edX101: Overview of Creating an edX Course'),
-            'text': _('Enroll in edX101'),
-            'condition': True
-        }, {
-            'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
-            'sr_mouseover_text': _('Enroll in StudioX: Creating a Course with edX Studio'),
-            'text': _('Enroll in StudioX'),
-            'condition': True
-        }, {
-            'href': 'mailto:{email}'.format(email=partner_email),
-            'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
-            'text': _('Contact Us'),
-            'condition': bool(partner_email)
-        }, {
-            'href': 'https://stanfordonline.zendesk.com/hc',
-            'sr_mouseover_text': _('Find answers in the FAQ or ask a new question.'),
-            'text': _('Help Center'),
-            'condition': True
-        }]
+          links = settings.STUDIO_SOCK_LINKS
```

Comment:
We don't need this variable, nor the entire code block.
Instead, just loop over the settings variable directly.

```
% for link in settings.STUDIO_SOCK_LINKS:
    <!-- whatever -->
% endfor
```